LondonBritishLibrary/orient/BLorient12859.xml

```diff
@@ -0,0 +1,439 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BLorient12859">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title>Magic scroll</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 12859</idno>
+                        <altIdentifier><idno>Or. 12859</idno></altIdentifier>
+                        <altIdentifier><idno>Strelcyn 80</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <title ref="LIT5888AsmatPrayer">Prayer for gaining respect</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ሀሊም፡ ወከመ፡ ቢስርቀያድ፡ በሥልጣነ፡ ፲ወ፬አስማተ፡ ርብሽመ፡ ከማሁ፡ ሙሮሸብአ፡
+                                ለሰብአ፡ ጋላ፡ ወለሻንቅላ፡ ወለሰብአ፡ ኵሉ፡ ፍጥረት፡ በግርማ፡ ገጸ፡ ገብርከ፡ <gap reason="lost"/></incipit>
+                            <note>Written in the left column under the third picture.</note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <title ref="LIT2393tabiba"/>
+                            <note>First stanza of the hymn. Written in the left column under the third picture.</note>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> እግዚአብሔር፡ ጠቢበ፡ ጠቢባን፡ ከሃሊ፤ እስመ፡ ለዓለመ፡ ዓለም፡ አሜን፡ እምጥንት፡ አንተ፡
+                                ኢትበሊ፤ ድኩመ፡ ሕሊና፡ ገብርከ፡ <del rend="effaced"/></incipit>
```

Comment:
If you agree that this is the better way of encoding this, the same must be applied in other places as well, I believe.

Reply:
Do you mean `<del rend="effaced"/>`? I do not know, if this is a deletion or an omission infact. I have not found any explanation for these brackets in the introduction or in the appendices of Strelcyns catalogue. Strelcyn may have used it for both, either for omissions or for deletions:

![grafik](https://github.com/user-attachments/assets/4342f8f4-9c4e-4b48-b5eb-6308a6784fb6)

Anyways, I think, it is small detail. I can change it here, if you wish, but I do not think, that it is necessary to revise a number of existing records on this detail. If a revision shall be done, it would be more reasonable to do it, the BL manuscripts are online again. But I have no information, when this will be the case.

Reply:
I meant only in this manuscript. Strelcyn writes that "The name of the owner has been erased", so I guess that when there is a space after ገብርከ፡ it's because of this. But it's a bit interpretative. Do as you find best :)

Reply:
Yes, I have done these changes in a new pull request.